### ISSUE TITLE
Using status bar view

### ISSUE THREAD

**Issue by noahbass** [yianL/atom-spotified#6]
Maybe I'm missing something, but after looking around I cannot get the status bar view shown in the README.md to display.

Is there a simple way to enable the status bar view?

Using v0.4.0 on atom 1.8.0 and OS X 10.11

**Comment by yianL** [comment]
Hi @noahbass, in the current implementation, status bar view will only appear when file tree is collapsed (`cmd+\`). How about we add a new option to the config:

``` js
mode = "AUTO" || "TREE" || "STATUSBAR"
// AUTO - current behavior
// TREE - tree view only
// STATUSBAR - status bar view only
```

**Comment by noahbass** [comment]
Ah I see now. Yes this option in the config would be great!

**Comment by yianL** [comment]
Fixed in `v0.5.0`. Cheers!

**Comment by noahbass** [comment]
Works great thanks!
